modules/custom/openy_upgrade_tool/src/OpenyUpgradeLogManager.php

```diff
@@ -0,0 +1,588 @@
+    return array_keys($openy_configs);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function isForceMode() {
+    return (bool) $this->configFactory
+      ->get('openy_upgrade_tool.settings')
+      ->get('force_mode');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function saveLoggerEntity($name, array $data, $message = NULL) {
+    try {
+      if ($this->getLoggerEntityTypeName() == 'logger_entity') {
+        // TODO: Delete this, 'logger_entity' is deprecated.
+        // Load logger entity with this config name.
+        $entities = $this->getLoggerEntityStorage()->loadByProperties([
+          'type' => 'openy_config_upgrade_logs',
+          'name' => $name,
+        ]);
+        if (empty($entities)) {
+          // Create new logger entity for this config name if not exist.
+          $logger_entity = $this->getLoggerEntityStorage()->create([
+            'type' => 'openy_config_upgrade_logs',
+          ]);
+        }
+        else {
+          $logger_entity = array_shift($entities);
+        }
+        $logger_entity->setName($name);
+        $logger_entity->setData([$name]);
+        $logger_entity->save();
+        return $logger_entity->id();
+      }
+      else {
+        // Load OpenyUpgradeLog entity with this config name.
+        $entities = $this->getLoggerEntityStorage()->loadByProperties([
+          'name' => $name,
+        ]);
+        if (empty($entities)) {
+          // Create new logger entity for this config name if not exist.
+          $upgrade_log_item = $this->getLoggerEntityStorage()->create([
+            'name' => $name,
+          ]);
+        }
+        else {
+          $upgrade_log_item = array_shift($entities);
+        }
+        $upgrade_log_item->setData($data);
+        $upgrade_log_item->setNewRevision(TRUE);
+        $upgrade_log_item->setStatus(FALSE);
+        $upgrade_log_item->setRevisionCreationTime(time());
+        $message = $message ?? 'Manual update outside Open Y.';
+        $upgrade_log_item->setRevisionLogMessage($message);
+        $upgrade_log_item->save();
+        return $upgrade_log_item->id();
+      }
+    }
+    catch (\Exception $e) {
+      $msg = 'Failed to save logger entity. Message: %msg';
+      $this->logger->error($msg, ['%msg' => $e->getMessage()]);
+      return FALSE;
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function createBackup($name) {
+    $config_data = $this->configStorage->read($name);
+    $revision_message = $this->t('Backup original config before force update.');
+    $this->saveLoggerEntity($name, $config_data, $revision_message);
+    return $this;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getLoggerEntityTypeName() {
+    if (!$this->moduleHandler || !$this->moduleHandler->moduleExists('logger_entity')) {
+      return 'openy_upgrade_log';
+    }
+
+    $logger_entity_exist = $this->entityTypeManager
+      ->getStorage('logger_entity_type')
+      ->load('openy_config_upgrade_logs');
```

Comment:
limit load by 100 the biggest, please

Reply:
Here you can see config entity load (not content) and it load entity by ID (in case config entity used name):
```php
  /**
   * Loads one entity.
   *
   * @param mixed $id
   *   The ID of the entity to load.
   *
   * @return \Drupal\Core\Entity\EntityInterface|null
   *   An entity object. NULL if no matching entity is found.
   */
  public function load($id);
```

As result we can get only 1 or 0 entities, so no need to add limit here.